src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,154 @@
-public class StringManipulation implements StringManipulationInterface {
+import java.util.ArrayList;
```

Comment:
Very weird that I encountered this too but only after the build was semi-successful.

I did not get the same error until I could actually build and have found that some of my test cases failed as well.

Good catch.

Reply:
Alvin, so I seemed to have misunderstood my own solution to the Travis build fail issue. The fundamental problem was my mishandling of the commit & add commands while commenting in/out the import statements. So, if I'm not mistaken again, StringManipulation() does in fact need the import statement for ArrayList and StringManipulationTest() should only have the original import statements from the skeleton, no others. 

After making sure this was the case, my build passed without issues (and without my panic), thank goodness.

In your case, I agree that your build had failed because, like you said, some of your test cases did not pass, and not because of import statement issues like me.